hotel/hotel/src/main/java/com/hotel/hotel/Model/GuestEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.hotel.hotel.Model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+@Entity
+@Data
+@Table(name = "guests")
+public class GuestEntity implements Serializable {
```

Comment:
I think lombok has an annotation to create a constructor with all arguments. can you check that out ?